src/store/store.ts

```diff
@@ -1170,18 +1170,23 @@ export const useStore = defineStore('store', {
       return !this.isRoleStaff && isDba
     },
 
-    getExpiryDateString (): string {
+    /**
+     * The restoration expiry date (YYYY-MM-DD).
+     * Only applicable to limited restoration extension filing.
+     */
+    getRestorationExpiryDate (): string {
       return this.stateModel.restoration?.expiry
     },
 
-    // FUTURE: should do something with "today"
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    getFormattedExpiryText: (state) => (today = new Date()): string => {
-      if (state.stateModel.restoration?.expiry) {
-        const numberOfExtensionMonths = DateUtilities.subtractDates(state.stateModel.stateFilingRestoration?.expiry,
-          state.stateModel.restoration?.expiry)
-        const expiryDatePacific = DateUtilities.yyyyMmDdToPacificDate(state.stateModel.restoration?.expiry)
-        return numberOfExtensionMonths + ' months, expires on ' + expiryDatePacific
```

Comment:
`today` wasn't being used, and this "getter as a function" was overly complicated, so I simplified it.